pkg/model/iam/iam_builder.go

```diff
@@ -212,6 +175,26 @@ func (b *IAMPolicyBuilder) BuildAWSIAMPolicy() (*IAMPolicy, error) {
 		}
 	}
 
+	p.Statement = append(p.Statement, &IAMStatement{
```

Comment:
Moving this is great - the code repetition is less acceptable given the route53 policies are no longer trivial.  But this is what is breaking the cloudformation test, because the order of the rules has changed.

A few options I think:

* Don't do the refactor
* Fix the test (it shouldn't be too bad, just rearranging the sections mostly)
* Pull this into a function and call it twice